### ISSUE TITLE
My Report does not work on the JasperReport Server in the Docker Container that I've spun up.

### ISSUE THREAD

**Issue by DaniyalAkbar** [bitnami/bitnami-docker-jasperreports#95]
**Description**

My Report does not work on the JasperReport Server in the Docker Container that I've spun up.

So I have cloned and run the command "docker-compose up -d". It successfully pulls the images and started the containers. I can also log into the Host_Port 8080 in my browser to have the JasperServer running just fine. The problem occurs when I run any report. I have created a JSON Data Adapter in JasperSoft Studio, which is fetching the JSON data that I've kept in another server and giving the Data Adapter the URL of the JSON file. It works fine in the Studio. But now I start the container and upload the JRXML file and the Data Adapter to the JasperReport Server which is running in the container. Now when I run the report it gives me an error. 

**The Output I Receive**
![image](https://user-images.githubusercontent.com/38786176/129318260-c5f6ca04-e504-461b-86b7-7df19a9d3871.png)

**What Output I Expect**
Since I've given it the Data Source which is the Remote JSON Data Adapter, it should fetch the data from the Remote Server and populate the report. 
I've already tested it working when I NOT use a docker container and instead manually installing JasperReport Server on my Windows laptop.

**Additional information you deem important (e.g. issue happens only occasionally):**
https://programatically.com/employees.json
Here is the JSON file URL from where it's fetching the data from.

When I check "docker-compose logs jasperreports", this is the error that shows up. 
![image](https://user-images.githubusercontent.com/38786176/129318924-c304dbef-40d7-44bc-9d64-626f8f610d15.png)

**Comment by pablogalegoc** [comment]
Hi @DaniyalAkbar!

> I’ve already tested it working when I NOT use a docker container and instead manually installing JasperReport Server on my Windows laptop.

Are you also running this image on Docker Desktop for Windows? Could you paste the output of `docker version`, `docker info` and `docker-compose version`?

**Comment by DaniyalAkbar** [comment]
yes, im running this image only on Docker Desktop for Windows, and that's when it's showing me the error. I'll just share the Docker info
![image](https://user-images.githubusercontent.com/38786176/129667133-ce549e2d-b3e6-4911-b167-b2e077cee5a1.png)
![image](https://user-images.githubusercontent.com/38786176/129667175-e07d1aa8-ef46-4939-ac28-dc6cbcbd32c0.png)

### This is the Docker version 
![image](https://user-images.githubusercontent.com/38786176/129667277-e905ea0d-fab8-4bb4-8302-a687382098ba.png)

### This is the Docker Compose version
![image](https://user-images.githubusercontent.com/38786176/129667320-3320993d-7351-4c76-8ed4-1d7f5a38f750.png)

**Comment by pablogalegoc** [comment]
Thanks, the last part in these logs seems to point to an issue with fonts. I’ve found some similar issues:
- [Getting an ExceptionInInitializerError on JREBaseFiller](https://community.jaspersoft.com/questions/1002016/getting-exceptionininitializererror-jrebasefiller)
- [Bold not working in Jaspersoft Studio for fonts other than sans serif](https://stackoverflow.com/questions/25977427/bold-not-working-in-jaspersoft-studio-for-fonts-other-than-sans-serif)

Seems that there are some incompatibilities with custom fonts and inconstancies with JasperSoft Studio

**Comment by DaniyalAkbar** [comment]
Okay so after the latest update on the bitnami repo, the error is gone and its working perfectly. 
But now I want to build my own image and run the container of only jasperreports. Not using docker-compose for this. 
in doing that, I'm getting an error when I run the container. When I check the logs through command "docker logs jasperreports", it shows the error as this:
### Error
![image](https://user-images.githubusercontent.com/38786176/130789420-176f1ef6-f66b-4f59-826d-05d5ce5a71aa.png)

I want to install the Webservice Datasource plugin in JasperReport Server. I'm following the setup from this link. https://community.jaspersoft.com/wiki/installing-webservice-datasource 

So, the only addition to your Dockerfile is a few COPY commands to copy the plugin files to WEB-INF folder of your container. ### This is the final version of my updated Dockerfile. 

![tempsnip](https://user-images.githubusercontent.com/38786176/130792081-1afa9918-4b68-48a1-b1ed-0deae15901b2.png)

 
After I build the image, I run the docker container using this command script:

docker run -d --name jasperreports \
  -p 8088:8089 -p 8443:8443 \
  --env ALLOW_EMPTY_PASSWORD=yes \
  --env JASPERREPORTS_DATABASE_USER=bn_jasperreports \
  --env JASPERREPORTS_DATABASE_PASSWORD=bitnami \
  --env JASPERREPORTS_DATABASE_NAME=bitnami_jasperreports \
  --env JASPERREPORTS_USERNAME=jasperadmin \
  --env JASPERREPORTS_PASSWORD=jasperadmin \
  --network jasperreports-network \
  --volume jasperreports_data:/bitnami/jasperreports \
  bitnami_image_webservice:latest
